### PR TITLE
Normalize Redux state shape

### PR DESCRIPTION
**Pull request by anniegiang**
- Setup `combineReducers` to achieve a normalized state shape
  - Moved the auth related reducer from `reducers/index.js` into `reducers/sessionReducer.js`, creating a  `session` slice of state
- Cleared some console warnings about unused variables
- Setup a basic `FeedContainer` to connect `Feed` with Redux
- Printed out the redux state 
  - I tried to setup `redux-logger` but couldn't get it to log in the console... so printing the state out is more of a bandaid fix

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ import * as serviceWorker from "./serviceWorker";
 import rootReducer from "./reducers";
 
 const store = createStore(rootReducer, applyMiddleware(thunk));
+console.log("redux state: ", store.getState());
```
Comment (robinv85) [inline client/src/index.js:17]: forgot log?

File: client/src/templates/RouteWithSubRoutes.js [hunk 2]
```diff
@@ -58,9 +58,9 @@ export const RouteWithSubRoutes = (route) => {
 };
 
 const mapDispatchToProps = {};
-const mapStateToProps = ({ emailVerified, isAuthenticated }) => ({
-  emailVerified,
-  isAuthenticated,
+const mapStateToProps = ({ session }) => ({
+  emailVerified: session.emailVerified,
+  isAuthenticated: session.isAuthenticated,
```
Comment (robinv85) [inline client/src/templates/RouteWithSubRoutes.js:63]: we might need to fine tune this a bit, it's not really a "session:" and the auth & user logic is a bit confusing at the moment but we can still improve that later I guess

File: client/src/index.js (hunk 1 above)
Comment (anniegiang) [reply client/src/index.js:17]: Left it there on purpose to temporary replace it with redux-logger.. but I can remove it for now.

File: client/src/templates/RouteWithSubRoutes.js (hunk 2 above)
Comment (anniegiang) [reply client/src/templates/RouteWithSubRoutes.js:63]: Ah ok sounds good

File: client/src/index.js (hunk 1 above)
Comment (robinv85) [reply client/src/index.js:17]: ah ok, i'm used to just check with redux-devtools if needed, i guess adding a logger could also work

**Pull request merged**